pkg/auth/user/util_test.go

```diff
@@ -0,0 +1,106 @@
+package user
+
+import (
+	"testing"
+	"time"
+
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/protoconv"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+)
+
+func TestExtractUserLogFields(t *testing.T) {
```

Comment:
👍 for adding unit tests. Can you split this into multiple tests, or at least a golang-style table test? In general I always opt to keep the number of assertions per test at a minimum.